### PR TITLE
Add "Navigate To Project" to project refs

### PR DESCRIPTION
**Pull request by davkean**
This exposes a new command, "Navigate To Project" on the right-click context menu of Project and Shared Project references that navigates to associated project in Solution Explorer. This is helpful for locating projects in large solutions with nested folders.

This includes https://github.com/dotnet/project-system/pull/6554, so just review https://github.com/dotnet/project-system/commit/88d5dc9055047df9d26d99dbf352f5231306ec5f.

NavigateToProject

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/project-system/pull/6559)

### CODE REVIEW

File: src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/Dependencies/ResolvedProjectReference.xaml [hunk 1]
```diff
@@ -25,6 +25,17 @@
...
     </StringListProperty.DataSource>
   </StringListProperty>
+  
+  <StringProperty Name="BrowsePath"
+                  ReadOnly="True"
+                  Visible="False">
+    <StringProperty.DataSource>
+      <DataSource ItemType="ProjectReference"
+                  PersistedName="Identity"
+                  Persistence="Intrinsic"
```
Comment (swesonga) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/Dependencies/ResolvedProjectReference.xaml:35]: >Intrinsic [](start = 31, length = 9)

What determines whether this is Intrinsic vs whether the attribute is excluded?

Comment (davkean) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/Dependencies/ResolvedProjectReference.xaml:35]: They are two different attributes; `Persistence` determines what backing "object" reads the value from the item and `PersistenceStyle` determines whether its rewritten out as an attribute or an element.

File: src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Packaging/ManagedProjectSystemPackage.cs [hunk 2]
```diff
@@ -22,7 +22,7 @@ namespace Microsoft.VisualStudio.Packaging
         termNames: new[] { "dotnetcore" },
         termValues: new[] { "SolutionHasProjectCapability:.NET & CPS" }
         )]
-    [ProvideMenuResource("Menus.ctmenu", 4)]
+    [ProvideMenuResource("Menus.ctmenu", 6)]
```
Comment (swesonga) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Packaging/ManagedProjectSystemPackage.cs:25]: >6 [](start = 41, length = 1)

curious, why isn't this just incremented to 5?

Comment (davkean) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Packaging/ManagedProjectSystemPackage.cs:25]: Just me showing how many times it took me to get right. :) 

I'll know it back to 5.

**Pull request merged**